src/main/webapp/app/exercises/shared/feedback/feedback-list.component.ts

```diff
@@ -0,0 +1,57 @@
+import { Component, Input } from '@angular/core';
+import { roundValueSpecifiedByCourseSettings } from 'app/shared/util/utils';
+import { AssessmentType } from 'app/entities/assessment-type.model';
+import { resultIsPreliminary } from 'app/exercises/shared/result/result.utils';
+import { faExclamationTriangle } from '@fortawesome/free-solid-svg-icons';
+import { Exercise } from 'app/entities/exercise.model';
+import { FeedbackItem, FeedbackItemType } from 'app/exercises/shared/result/result-detail.component';
+import { Result } from 'app/entities/result.model';
+import { Course } from 'app/entities/course.model';
+
+@Component({
+    selector: 'jhi-feedback-list',
+    templateUrl: './feedback-list.component.html',
+    styleUrls: ['./feedback-list.scss'],
+})
+export class FeedbackListComponent {
+    readonly AssessmentType = AssessmentType;
+    readonly roundValueSpecifiedByCourseSettings = roundValueSpecifiedByCourseSettings;
+    readonly resultIsPreliminary = resultIsPreliminary;
+    readonly FeedbackItemType = FeedbackItemType;
+
+    @Input() exercise?: Exercise;
```

Comment:
I will try to address this in the follow up by further refactoring logic into `services`